src/test/java/BaseTestIvan.java

```diff
@@ -0,0 +1,165 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.openqa.selenium.safari.SafariDriver;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.DataProvider;
+
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URL;
+import java.time.Duration;
+
+public class  BaseTestIvan {
+
+    public static WebDriver driver;
+    public static WebDriverWait wait;
+    public static String url;
+    public static Actions actions;
+    public static ThreadLocal<WebDriver> threadDriver;
+
+
+
+    @BeforeSuite
+    public static void chromeConfigs() {
+        // This is for Windows users
+        if (System.getProperty("os.name").toLowerCase().contains("win")) {
+            System.setProperty("webdriver.chrome.driver", "chromedriver.exe");
+        } else {
+            System.setProperty("webdriver.chrome.driver", "chromedriver");
+        }
+    }
+//    @BeforeMethod
+//    public static void launchBrowser() throws MalformedURLException {
+//        System.setProperty("webdriver.gecko.driver", "geckodriver");
+//        //driver = new ChromeDriver();
+//        //driver = new FirefoxDriver();
+//        //driver = new SafariDriver();
+//        threadDriver = new ThreadLocal<>();
+//        driver = pickBrowser(System.getProperty("browser"));
+//        threadDriver.set(driver);
+//        //actions = new Actions(getDriver());
+//        //driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+//        wait = new WebDriverWait(getDriver(), Duration.ofSeconds(10));
+//        url = "https://bbb.testpro.io/";
+//        getDriver().get(url);
+//
+//
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.